### PR TITLE
Adding more functional tests for blank-raw-image, PV allocation on specific node and upload disk image features

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: tavni <tavni@redhat.com>

**What this PR does / why we need it**:
we need more functional tests to increase our testing coverage.

**Which issue(s) this PR fixes** :
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: tests/datavolume_test.go [hunk 1]
```diff
@@ -128,6 +126,73 @@ var _ = Describe("DataVolume tests", func() {
...
+				}
+				fmt.Fprintf(GinkgoWriter, "ERROR: %s\n", err.Error())
+				return false
+			}, timeout, pollingInterval).Should(BeTrue())
+
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, dataVolume.Name)
+			Expect(err).ToNot(HaveOccurred())
+
+		},
+			table.Entry("succeed creaing blank-image dv", cdiv1.Succeeded, "blank-image-dv", controller.ImportSucceeded),
```
Comment (mhenriks) [inline tests/datavolume_test.go]: looks like these tests (new and existing) are very similar.  It is also kind of weird that they are table driven tests that have only one Entry.  Does it make sense to combine them into a single table driven test with multiple Entrys?

File: tests/upload_test.go [hunk 2]
```diff
@@ -60,4 +61,45 @@ var _ = Describe("Upload tests", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(deleted).To(BeTrue())
 	})
+
+	It("Upload to PVC should fail", func() {
```
Comment (mhenriks) [inline tests/upload_test.go]: Slight tweak to "success" test.  Maybe table driven makes sense?

File: tests/local_volume_test.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	"fmt"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+	"time"
+)
+
+const (
+	endPoint         = "http://distro.ibiblio.org/tinycorelinux/9.x/x86/release/Core-current.iso"
```
[the remark this answers is not in the document]
Comment (zvikorn) [reply tests/local_volume_test.go]: done.

File: tests/datavolume_test.go (hunk 1 above)
Comment (zvikorn) [reply tests/datavolume_test.go]: done

File: tests/upload_test.go (hunk 2 above)
Comment (zvikorn) [reply tests/upload_test.go]: done

**Comment by igoihman** [comment]
ci test please

**Pull request merged**